api/types.proto

```diff
@@ -781,33 +781,36 @@ message ManagerStatus {
 	RaftMemberStatus.Reachability reachability = 4;
 }
 
+// File represents a specific target that is backed by a file
+message File {
```

Comment:
`FileMetadata`? Or define this inside `SecretReference` so it doesn't pollute the namespace.